packages/rich-text/src/component/index.native.js

```diff
@@ -18,6 +18,7 @@ import {
 } from '@wordpress/react-native-bridge';
 import { BlockFormatControls, getPxFromCssUnit } from '@wordpress/block-editor';
 import { Component } from '@wordpress/element';
+import { KEYCODES_MOBILE } from '@wordpress/components';
```

Comment:
What is the rationale for placing this in a separate location, i.e. `@wordpress/components`? It appears this only used in Rich Text currently. Would we benefit from colocating it here until it is required elsewhere?

Reply:
I did it like that as we always do it for any mobile utils we add. But like you mentioned since this is only used in the Rich Text component I'll move it to the AztecView component instead as mentioned in https://github.com/WordPress/gutenberg/pull/43521#discussion_r954012735

We can always move it back if we need it somewhere else in the codebase 👍 

Thanks!